mangaki/mangaki/utils/data.py

```diff
@@ -37,6 +40,17 @@ def load(self, filename):
         self.decode_work = backup.decode_work
         self.interesting_works = backup.interesting_works
 
+    def load_csv(self, filename, convert=float):
+        with open(os.path.join(settings.BASE_DIR, '../data', filename)) as f:
```

Comment:
Why not write a `CSV_DIR` inside of `settings`? Let's standardize this directory across our codebase.